intake_bluesky/core.py

```diff
@@ -308,15 +403,16 @@ class BlueskyRun(intake.catalog.Catalog):
         Expected signature ``get_run_stop() -> RunStop``
     get_event_descriptors : callable
         Expected signature ``get_event_descriptors() -> List[EventDescriptors]``
-    get_event_cursor : callable
-        Expected signature ``get_event_cursor(descriptor_uids) -> generator``
+    get_event_pages : callable
+        Expected signature ``get_event_pages(descriptor_uids) -> generator``
         where ``generator`` yields Event documents
     get_resource : callable
         Expected signature ``get_resource(resource_uid) -> Resource``
-    get_datum : callable
-        Expected signature ``get_datum(datum_id) -> Datum``
-    get_datum_cursor : callable
-        Expected signature ``get_datum_cursor(resource_uid) -> generator``
+    lookup_resource_for_datum : callable
+        Expected signature ``lookup_resource_for_datum(datum_id) ->
```

Comment:
This formatting doesn't work across a line break, so just put a new line after `signature`.

Reply:
updated